Tasks/UseRubyV1/Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
From https://github.com/Microsoft/azure-pipelines-tasks/pull/9875#discussion_r269715936, we should leave out the loc files for now.

Reply:
Removed `Strings/`